x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/original_scenarios.md

```diff
@@ -0,0 +1,264 @@
+When <case>
+Then the available rules state should persist for all the rules
+
+Examples:
+  | case                              |
+  | user reloads the page             |
+  | after switching table pagination  |
+  | after filtering and clear filters |
+```
+
+#### **Scenario: User can navigate back to Rules Management page**
+
+```Gherkin
+Given a user is on Add Rules Page
+When user navigates back to Rules Management page
+Then Rules Management Page is properly displayed
+```
+
+#### **Scenario: User can filter prebuilt rules by rule name or by tag**
+
+```Gherkin
+Given a user is on Add Rules Page
+When user filters by <filter>
+Then Add Rules Table is properly updated
+
+Examples:
+  | filter                  |
+  | rule name on search bar |
+  | Tag filter              |
+```
+
+### Rule Updates tab
+
+#### **Scenario: Rules settings persist on Rule Updates table**
+
+```Gherkin
+Given a user has Rules listed on Rule Updates table
+When <case>
+Then the rules with available updates state should persist
+
+Examples:
+  | case                              |
+  | user reloads the page             |
+  | after switching table pagination  |
+  | after filtering and clear filters |
+```
+
+#### **Scenario: User can navigate back to Rules Management tab**
+
+```Gherkin
+Given a user is on Rule Updates tab
+When user navigates back to Rules Management page
+Then Rules Management Page is properly displayed
+```
+
+#### **Scenario: User can filter prebuilt rules by rule name or by tag**
+
+```Gherkin
+Given a user is on Rule Updates tab
+When user filters by <filter>
+Then Rule Updates tab is properly updated
+
+Examples:
+  | filter                  |
+  | rule name on search bar |
+  | Tag filter              |
+```
+
+### Authorization / RBAC
+
+#### **Scenario: User with read privileges on security solution cannot install prebuilt rules**
+
+```Gherkin
+Given a user with Security: read privileges on Security solution
+When user navigates to Add Elastic Rules Page
+Then user can see available prebuilt rules to install
+And user cannot Install those prebuilt rules
+```
+
+#### **Scenario: User with read privileges on security solution cannot update prebuilt rules**
+
+```Gherkin
+Given a user with Security: read privileges on Security solution
+When user navigates to Rule Updates Tab on Rules Management Page
+Then user can see new updates for installed prebuilt rules
+And user cannot Update those prebuilt rules
+```
+
+### Kibana upgrade
+
+#### **Scenario: User can operate with prebuilt rules when user upgrades from version `<version>` to 8.9 version**
+
+```Gherkin
+Given a user that is upgrading from version <version> to version 8.9
+And the <version> instance contains already installed prebuilt rules
+When the upgrade is complete
+Then user can install new prebuilt rules
+And remove installed prebuilt rules
+And update prebuilt rules from <version> to 8.9
+
+Examples:
+  | version |
+  | 8.7     |
+  | 7.17.x  |
+```
```

Comment:
Replaced by:

#### **Scenario: User can use prebuilt rules after upgrading Kibana from version A to B**

**Coverage**: not covered, manual testing required.

```Gherkin
Given user is upgrading Kibana from version <A> to version <B>
And the <A> instance contains already installed prebuilt rules
When the upgrade is complete
Then user should be able to install new prebuilt rules
And delete installed prebuilt rules
And upgrade installed prebuilt rules that have newer versions in <B>

Examples:
  | A      | B     |
  | 8.7    | 8.9.0 |
  | 7.17.x | 8.9.0 |
```